src/css/imports/controlbar.less

```diff
@@ -2,7 +2,7 @@
 @import "vars";
 
 .jw-controlbar {
-  display: inline-block; // for correct display in IE browsers
+  display: table;
```

Comment:
Since you changed this back to `table`, you can also remove the rule specific to time-slider-above:

```
.jw-flag-time-slider-above:not(.jw-flag-ads-googleima) {

  .jw-controlbar {
    display: table;
  }
```